integration-tests-scala/src/test/java/tech/beshu/ror/integration/Proxy.java

```diff
@@ -0,0 +1,11 @@
+package tech.beshu.ror.integration;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@org.scalatest.TagAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface Proxy {}
```

Comment:
I needed to tag these tests somehow because I couldn't exclude/include tests by name or package for gradle runner... Now I'm using `suite` in `build.gradle` and it seems  tests filtering by package works fine, no tags needed. Removed